Snowflake.Data.Tests/SFConnectionIT.cs

```diff
@@ -59,7 +62,7 @@ public void TestConnectViaSecureString()
                 conn.ConnectionString = connectionStringWithoutPassword;
                 conn.Password = password;
                 conn.Open();
-                Assert.AreEqual("TESTDB_DOTNET", conn.Database);
+                Assert.AreEqual(testConfig.database, conn.Database);

```

Comment:
The database is configurable but here fixed database name was assumed. Check against configs instead.